pkg/csi/service/wcp/controller.go

```diff
@@ -148,6 +147,19 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 						time.Sleep(5 * time.Second)
 					}
 				}
+				if event.Op&fsnotify.Create == fsnotify.Create {
+					if event.Name == cnsconfig.SupervisorCAFilePath {
```

Comment:
Can you provide note about why we are watching on the `fsnotify.Create` event?

Reply:
+1  on this.

Reply:
Added